templates/openstackclient/bin/init.sh

```diff
@@ -69,3 +60,59 @@ EOF
   git config --global user.email "dev@null.io"
   git config --global user.name "OSP Director Operator"
 fi
+
+
+# Skip the rest on tempest pod
+if cat /etc/hostname | grep ^tempest 2>&1 > /dev/null; then exit 0; fi
```

Comment:
instead of grep for tempest hostname, should we add a parameter to openstackclient and pass the info via a env var, so this part can be skipped on CR base when we create the tempest pod?

Reply:
Yes, this is just a quick and dirty hack to get tempest working. The tempest user isn't in sudoers so many of the init commands do not apply.

Reply:
Actually I think I can just drop that line - it was added before the next block was conditional on IPA_SERVER which will not be set for the tempest pod.

Reply:
> Actually I think I can just drop that line - it was added before the next block was conditional on IPA_SERVER which will not be set for the tempest pod.

testing this now...

Reply:
works with that line removed